test/itemlist_formaction.cpp

```diff
@@ -0,0 +1,154 @@
+#include "catch.hpp"
+
+#include "cache.h"
+#include "itemlist_formaction.h"
+#include "keymap.h"
+#include "stfl/itemlist.h"
+
+#include <unistd.h>
+
+using namespace newsbeuter;
+
+TEST_CASE("itemlist_formaction::process_op()") {
+
+	controller c;
+	newsbeuter::view v(&c);
+	std::string test_browser_file = "/tmp/test_browser.txt";
+	std::string test_url = "http://test_url";
+	std::string cache_path = "test-cache.db";
+	std::string line;
+	int itemCount = 6;
+	int openedItemsCount = 0;
+	int lineCount = 0;
+	int maxItems = 4;
+	int openedItemsTarget = 0;
+
+	configcontainer * cfg = new configcontainer();
+	cfg->set_configvalue("browser", "echo %u >> " + test_browser_file);
+	cfg->set_configvalue("max-browser-tabs", std::to_string(maxItems));
```

Comment:
This line belongs to the first test case, because the rest sets `max-browser-tabs` themselves.